### ISSUE TITLE
Smartclient não abre ao DEBUGAR - VS-CODE

### ISSUE THREAD

**Issue by SamuelRenan** [totvs/tds-vscode#1170]
Ao iniciar o debug pelo Visual Studio o mesmo não inicia, só carrega infinitamente, no terminal e console não aparece nada, fica em branco, como nas imagens abaixo

![image](https://github.com/totvs/tds-vscode/assets/103660872/11de74f8-a2a4-4dcb-b3fd-757e5dd4f0c4)

![image](https://github.com/totvs/tds-vscode/assets/103660872/bd3c557f-b8da-40b7-958a-6ca1fdbaaa1e)

![image](https://github.com/totvs/tds-vscode/assets/103660872/3e65158b-625f-4f6c-8695-475b8c742251)

Esse erro ja vem acontecendo de um tempo, as vezes ele se resolve sozinho, mas na maioria das vezes permanece assim
Já reiniciei os serviços, executei o AppServer como console e nada....

**Comment by lwtnb-wrk** [comment]
@SamuelRenan Veja em https://github.com/totvs/tds-vscode/blob/master/TROUBLESHOOTING.md#gerando-logs como gerar e nos envie os logs para analises. Em especial o dap.log por se tratar de depuracao.

**Comment by SamuelRenan** [comment]
Boa tarde, acrescentei as chaves que estava no documento, porém o debug não inicia mais e nem gerou os logs solicitados

![image](https://github.com/totvs/tds-vscode/assets/103660872/95ae4643-5c53-49ac-940b-58d15e42b739)

**Comment by lwtnb-wrk** [comment]
@SamuelRenan Tira a última virgula "," pois isso deve ter causado a quebra no arquivo de configuração.

**Comment by SamuelRenan** [comment]
que ultima virgula seria?


Em seg., 4 de dez. de 2023 às 15:49, lwtnb-wrk ***@***.***>
escreveu:

**Comment by SamuelRenan** [comment]
![image](https://github.com/totvs/tds-vscode/assets/103660872/245a32ae-8c18-4f17-87fa-54f553fe53af)

**Comment by lwtnb-wrk** [comment]
@SamuelRenan No final da linha que você adicionou... Tem que ser apenas:
"logFile": "${workspaceFolder}\\dap.log"
sem a virgula no final da linha.

**Comment by lwtnb-wrk** [comment]
@SamuelRenan No "totvsLanguageServer.launch.args" precisa da virgula, pois tem outra linha depois

**Comment by SamuelRenan** [comment]
a virgula deve ir depois dos dois pontos?

**Comment by lwtnb-wrk** [comment]
@SamuelRenan Alias essa parte não deveria estar no arquivo launch.json conforme está descrito na documentação.

**Comment by SamuelRenan** [comment]
o correto seria apenas essas duas linhas?

"--log-file=totvsls.log"
		"--record=totvsls"

**Comment by SamuelRenan** [comment]
peço desculpas, acabei de perceber que estava no arquivo errado, irei jogar no certo e rodar o debug para gerar os logs.

**Comment by lwtnb-wrk** [comment]
@SamuelRenan Ao inves de depurar, voce consegue iniciar o Smartclient atraves do CTRL + F5 ?

**Comment by lwtnb-wrk** [comment]
@SamuelRenan Poderia nos enviar seu launch.json ? Aparentemente ele está corrompido.

**Comment by SamuelRenan** [comment]
[launch.json](https://github.com/totvs/tds-vscode/files/13560523/launch.json)

**Comment by lwtnb-wrk** [comment]
@SamuelRenan Aparentemente tem algum problema na substituição de uma variável e vamos investigar. 
Para contornar isso, por enquanto, utilize uma outra pasta para colocar seus fontes, de preferencia uma pasta sem espaços e acentuações em todo o seu caminho.

**Comment by SamuelRenan** [comment]
Certo, mas o problema está mesmo quando não tem nenhum fonte na pasta, e sem acentuação ou espaço no nome da pasta.

**Comment by lwtnb-wrk** [comment]
@SamuelRenan É isso que falei sobre a acentuação no nome da pasta. De qualquer forma vamos avaliar se conseguimos tratar isso e não ter problema com isso no futuro.
